src/site/static/eligibility.liquid

```diff
@@ -224,7 +408,7 @@ pageClass: "page-public-assistance"
   // This global variable holds the current state of the form navigation.
   let currentPage;
 
-  // Shows or hides the element 'elem' via a class name.
+// Shows or hides the element 'elem' via a class name.
```

Comment:
Is it possible to undo these auto-formatter changes?  Currently, it makes it a bit hard to see actually which code was changed.